sporks.json

```diff
@@ -1309,6 +1309,45 @@
       }
     },
     "testnet": {
+      "testnet48": {
+        "id": 48,
+        "live": true,
+        "name": "Testnet48",
+        "sporkTime": "2023-10-25T14:30:00Z",
+        "rootHeight": "127720465",
```

Comment:
this should be `127720466` according to the root snapshot
https://storage.googleapis.com/flow-genesis-bootstrap/testnet-48/public-root-information/root-protocol-state-snapshot.json